### PR TITLE
Add Nord theme

### PR DESCRIPTION
**Pull request by JayBigGuy10**
I had no idea how to add to my last merge request so I closed them and opened this one with both files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@
 | Sublimination   | [MikaelSchultz](https://community.home-assistant.io/u/MikaelSchultz) posted [Sublimination Theme](https://community.home-assistant.io/t/sublimination-theme/67312)   | sublimination       |
 | Black and Green | [GreenTurtwig](https://community.home-assistant.io/u/GreenTurtwig) posted [Black and Green Theme](https://community.home-assistant.io/t/black-and-green-theme/28602) | teal                |
 | Vintage         | [surendran.anup](https://community.home-assistant.io/u/surendran.anup) posted [Vintage Theme](https://community.home-assistant.io/t/vintage-theme/42806)             | vintage             |
+| Nord            | [Jasperro](https://community.home-assistant.io/u/jasperro) posted [Nord Theme](https://community.home-assistant.io/t/nord-theme/129301)             | nord                |
```
Comment (maartenpaauw) [inline README.md]: Could you please leave this list alphabetical ordered (based on theme name)?

File: nord.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  paper-font-common-base_-_font-family: "var(--primary-font-family)"
+  paper-font-common-code_-_font-family: "var(--primary-font-family)"
+  paper-font-body1_-_font-family: "var(--primary-font-family)"
+  paper-font-subhead_-_font-family: "var(--primary-font-family)"
+  paper-font-headline_-_font-family: "var(--primary-font-family)"
+  paper-font-caption_-_font-family: "var(--primary-font-family)"
+  paper-font-title_-_font-family: "var(--primary-font-family)"
+  ha-card-header-font-family: "var(--primary-font-family)"
+  ha-card-header-color: "var(--foreground-color)"
+  sidebar-icon-color: "var(--foreground-color)"
```
Comment (maartenpaauw) [inline nord.yaml:48]: Is there an extra blank line at the end of this file?

File: README.md (hunk 1 above)
Comment (maartenpaauw) [inline README.md]: Could you please leave this list alphabetical ordered (based on theme name)?

**Review by maartenpaauw — CHANGES_REQUESTED** [review]

**Comment by maartenpaauw** [comment]
Thanks for combining the pull requests. I left 2 comments on the code. If you could fix this I'm happy to merge the PR.

**Comment by JayBigGuy10** [comment]
Hi I have updated the files

**Review by maartenpaauw — APPROVED** [review]

**Pull request merged**